### PR TITLE
Improve current EXIF processing

### PR DESCRIPTION
**Pull request by Sammidysam**
This is largely a precursor to the next steps on my end, which is implementing EXIF processing within our main Community Voices architecture (done; see `exif-server` branch) and then trying to use a JavaScript library to process EXIF data so that the server does nothing at all for this (next step for me).

The two (listed as three bullet points, but two things) main changes in this pull request:
- Allow local JavaScript development by pulling a relative source for `extraJS` as opposed to forcing to pull from `environmentaldashboard.org` - This is a hugely important change that I plan to extend to CSS etc.
- Local reference `public/exif.php` for accessing EXIF data from the server.
- I really wanted to fix all of these images having their date set to 0, which ends up being 1969 (see https://www.environmentaldashboard.org/community-voices/images/4204 as an example; created at is set correctly but its specific date in the admin photo view is so bad).  So, I decided on this stream of logic for what to set the photo date to by default:
If `exif.DateTime` exists, this photo was likely taken by a camera (it seems that cameras set that field).  So, we want to use that field if it exists for sure.
If it does not, the file may have some creation date attached to it.  This could be `exif.FileDateTime`.  This is less ideal, but still a good estimate of when it was taken.
Finally, if neither of these exist, I would say the current time of upload is a better estimate of when the photo was taken than 1969 (time = 0).
Maybe it's worth going in and reassigning some of these image dates in the database too, honestly.  Just scroll around https://www.environmentaldashboard.org/community-voices/images to see what I mean.

Basically, while working today I realized there's no reason this branch shouldn't be integrated asap; so just wondering of your thoughts on the changes.

### CODE REVIEW

**Comment by Sammidysam** [comment]
New question too: do you like the setting of `var date` better in one line or in a paragraph?  I am torn honestly.  Reference: https://github.com/EnvironmentalDashboard/community-voices/pull/16/commits/0e660dc2c4b8f981ea0cb4b77e6b8a20ddcb7d49

**Comment by Terf** [comment]
I think leaving an images date taken as `0` is better than lying about when it was taken, we would just not show the date on the front end then (rather than showing 1969).

**Comment by Sammidysam** [comment]
That's a good point.  We do have a different created at date too, which we could show as a fall back to our official date variable.

**Comment by Sammidysam** [comment]
I can adjust this that way before we merge.
